standard-deployment.cfg

```diff
@@ -20,6 +21,7 @@ parts -=
 usernamelogger_ac_cookie_name = __ac
 raven_project_dist = opengever.core
 slack-webhook = $STANDARD_SLACK_WEBHOOK
+maintenance-directory = ${buildout:directory}/src/opengever.maintenance/maintenance-page
```

Comment:
page has been added in https://github.com/4teamwork/opengever.maintenance/pull/65